### PR TITLE
Update cayenne-guide.html minor text edits lines 461-1000

### PR DESCRIPTION
**Pull request by cottage14**

### CODE REVIEW

**Comment by andrus** [comment]
The PR is submitted against the website repo. HTML documentation files here are compiled from Asciidoc source files located elsewhere at https://github.com/apache/cayenne/tree/master/docs/asciidoc . So we should not be changing them directly. A PR against the Asciidoc folder above would be great

File: src/main/site/content/docs/4.2/cayenne-guide.html [hunk 1]
```diff
@@ -524,22 +524,22 @@ <h4 id="objectcontext"><a class="anchor" href="#objectcontext"></a>2.3.1. Object
...
     </div> 
     <div class="paragraph"> 
-     <p>Persistent interface provides access to 3 persistence-related properties - <em>objectId</em>, <em>persistenceState</em> and <em>objectContext</em>. All 3 are initialized by Cayenne runtime framework. Application code should not attempt to change them. However it is allowed to read them, which provides valuable runtime information. E.g. ObjectId can be used for quick equality check of 2 objects, knowing persistence state would allow highlighting changed objects, etc.</p> 
+     <p> The persistent interface provides access to three persistence-related properties - <em>objectId</em>, <em>persistenceState</em> and <em>objectContext</em>. All three are initialized by the Cayenne runtime framework. Your application code should not attempt to change them. However, it is allowed to read them, which provides valuable runtime information. E.g. ObjectId can be used for a quick equality check of two objects, knowing persistence state would allow highlighting changed objects, etc.</p>
```
Comment (andrus) [inline src/main/site/content/docs/4.2/cayenne-guide.html:542]: Here "Persistent" is the name of the interface, so it should be capitalized (and probably wrapped in <code> block)

**Comment by kaganem** [comment]
Just created https://github.com/apache/cayenne/pull/590 with edits moved to ASCIIdoc source. @cottage14 may you have a look?

**Comment by andrus** [comment]
I merged https://github.com/apache/cayenne/pull/590 . So this one needs to be closed.

**Pull request closed without merging**